src/cf-tables/src/cf-table-sortable.js

```diff
@@ -0,0 +1,206 @@
+
+/* ==========================================================================
+   Table Sortablle
+
+   Mixin for sorting table organism.
+
+   ========================================================================== */
+
+'use strict';
+
+var config = require( 'atomic-component/src/utilities/config' );
+var closest = require( 'atomic-component/src/utilities/dom-closest' ).closest;
+var DIRECTIONS = config.DIRECTIONS;
+var UNDEFINED = config.UNDEFINED;
+
+var TableSortable = {
+
+  classes: {
+    sortDown: 'sorted-down',
+    sortUp:   'sorted-up'
+  },
+
+  events: {
+    'click .sortable': 'onSortableClick'
+  },
+
+  ui: {
+    base:       '.o-table__sortable',
+    tableBody:  'tbody',
+    sortButton: '.sorted-up, .sorted-down'
+  },
+
+  /**
+   * Function used to create computed and triggered properties.
+   */
+  initialize: function() {
+    this.sortClass = UNDEFINED;
+    this.sortColumnIndex = UNDEFINED;
+    this.sortDirection = UNDEFINED;
+    this.tableData = [];
+    this.bindProperties();
+    if ( this.ui.sortButton ) {
```

Comment:
It gets overwritten with an actual DOM element if it exists, otherwise it's null.